stdlib/public/core/StringGuts.swift

```diff
@@ -90,6 +90,11 @@ extension _StringGuts {
     @inline(__always) get { return _object.isSmall }
   }
 
+  @inlinable
```

Comment:
This doesn't need to be `@inlinable` anymore, right?

Reply:
No, though everything inside it already is inlinable. I'll drop it